src/lib.rs

```diff
@@ -165,6 +165,8 @@
 //!   buffer or `Reader` (thanks to the change of `Value::Undefined`
 //!   and `Value::Ascii` described above).
 
+#![forbid(unsafe_code)]
```

Comment:
Any reason why unsafe code should be forbidden?

Reply:
If you don't *use* unsafe code, then you should forbid it, as it makes cargo geiger's job easier, it makes it clearer that you're not using unsafe anywhere, and there are some things that are unsafe but you don't put an `unsafe` keyword around, and forbid(unsafe_code) forbids those.

Of course, if you do need unsafe code for a good reason, you can't do that.